src/Radio/APRS/APRSTelemMsg.h

```diff
@@ -0,0 +1,43 @@
+#ifndef APRS_TELEM_MSG_H
+#define APRS_TELEM_MSG_H
+
+#include "APRSMsgBase.h"
+#include <imumaths.h>
```

Comment:
makes reliance on having the whole imumaths library somewhere which may want to be avoided idk

Reply:
I agree. We should rewrite this library ourselves or at the very least include it on its own without the Adafruit library in the lib folder. (I think this used to be done, but I may have removed it without understanding why it was there)

Reply:
I agree it would be useful to have our own math library, but that's something that should be done as part of a separate issue since lots of our code relies on ```imumaths.h```.